src/python/grpcio/grpc/compatibility.py

```diff
@@ -0,0 +1,6 @@
+import six
+
+if six.PY2:
```

Comment:
This module smells like it doesn't have to exist. Why would `six.moves.cStringIO` not work in places where `grpc.compatibility.StringIO` is drafted for use?